packages/grant-explorer/src/features/collections/CollectionsGrid.tsx

```diff
@@ -0,0 +1,35 @@
+import CollectionCard from "./CollectionCard";
+import { Collection } from "./hooks/useCollections";
+
+export function CollectionsGrid({
+  isLoading,
+  data,
+  maxCount,
+  itemClassName,
+}: {
+  isLoading?: boolean;
+  data?: Collection[];
+  loadingCount: number;
+  maxCount?: number;
+  itemClassName?: (collection: Collection, index: number) => string;
+}) {
+  if (!isLoading && !data?.length) {
+    return <CollectionsEmptyState />;
+  }
+
+  return (
+    <div className="grid md:grid-cols-4 gap-6">
+      {data?.slice(0, maxCount).map((collection, i) => (
+        <div key={collection?.id} className={itemClassName?.(collection, i)}>
```

Comment:
Can `collection` actually be `undefined` at this point?

Also, I had read `itemClassName` as value and was surprised when it was called. A name that starts with a verb (such as `getItemClassName`) would make it clear that it is a function.

Reply:
Thanks, fixed these.